setup.py

```diff
@@ -75,19 +75,20 @@ def read_file(name):
     'django_polymorphic==0.6.1',
     'dnspython',
     'fabric',
-    'django-tenant-schemas==1.5.2',
+    'django-tenant-schemas==1.5.7',
     'raven==5.1.1',
     'regex==2015.11.22',
     'djangorestframework-jwt==1.7.2',
     'django-filetransfers==0.1.0',
     'django-admin-tools==0.5.2',
     'django-rest-swagger==0.3.4',
+    'django-webtest==1.7.8',
```

Comment:
This should probably only be in tests_requires. If not, the version should match.